catroid/src/androidTest/java/org/catrobat/catroid/test/io/filepicker/ListProjectFilesTest.java

```diff
@@ -90,4 +101,35 @@ public void testListFiles() throws IOException {
 		assertThat(projectFiles, not(hasItem(file2)));
 		assertThat(projectFiles, not(hasItem(file4)));
 	}
+
+	@Test
+	public void testListProjectsOnExternalStorage() throws IOException {
+
+		File backpackFolder = new File(EXTERNAL_STORAGE_ROOT_DIRECTORY, "backpack");
+		assertTrue(backpackFolder.mkdirs());
+
+		File tempFolder = new File(EXTERNAL_STORAGE_ROOT_DIRECTORY, Constants.TMP_DIR_NAME);
```

Comment:
it would be nice if you could change it to static imports for the constants. 

>"So when should you use static import? [..] In other words, use it when you require frequent access to static members >from one or two classes.  [..] "

from [Oracle doc](https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html). besides, if want to move the constants (e.g. to `FlavoredConstants`) then you don't have to make sure that every occurrence is changed correctly.